vital/types/detected_object.h

```diff
@@ -105,10 +127,10 @@ class VITAL_EXPORT detected_object
    * @brief Get bounding box from this detection.
    *
    * The bounding box for this detection is returned. This box is in
-   * image coordinates. A default constructed (invalid) bounding box
```

Comment:
(**Documentation**) Uh... I think this change made sense when the bbox was a pointer, and actually does *not* make sense now... (Although "A default constructed (invalid)" → "An invalid" might be good.) Also, if you end up making a change, please re-wrap this and the following line(s)¹.

The @return change makes sense though.

(¹ "Lines" in the general case; in this case there is only one.)